go.mod

```diff
@@ -121,6 +121,9 @@ replace (
 
 	go.universe.tf/metallb => github.com/cilium/metallb v0.1.1-0.20210607221240-b4c60b959dd7
 
+	// Make sure to use grpc v1.29.1 as grpc/naming package was removed in newer versions.
+	google.golang.org/grpc => google.golang.org/grpc v1.29.1
```

Comment:
This was added due to:
```
go: finding module for package google.golang.org/grpc/naming
github.com/cilium/cilium/pkg/kvstore imports
	go.etcd.io/etcd/clientv3 tested by
	go.etcd.io/etcd/clientv3.test imports
	go.etcd.io/etcd/integration imports
	go.etcd.io/etcd/proxy/grpcproxy imports
	google.golang.org/grpc/naming: module google.golang.org/grpc@latest found (v1.47.0), but does not contain package google.golang.org/grpc/naming
```

Perhaps better would be to bump etcd here as well? Thoughts?

Reply:
Probably fine to keep the grpc lib pinned here. This is a v1.10 branch which we won't typically expect changes to vendor moving forward.